core/examples/luigi-sample-angular/src/assets/basicexternal.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Basic example</title>
+  <link rel='stylesheet' href='/fundamental-ui/fundamental-ui.min.css'> 
+  <style type="text/css">
+  body {
+    font-family: Arial, Helvetica, sans-serif;
+  }
+  .section-spacer {
+    height: 800px;
+  }
+  .fd-panel {
+    padding: 15px;
+  }
+  a {
+    color: #0a6ed1;
+    text-decoration: none;
+  }
+  a:hover {
+    color: #085caf;
+  }
+  </style>
+</head>
+<body>
+<a name="ext"></a>
+<header class="fd-page__header">
+  <div class="fd-action-bar">
+    <div class="fd-action-bar__header">
+      <h1 class="fd-action-bar__title">
+          Basic Example page
+      </h1>
+      <p class="fd-action-bar__description">This page behaves equally to an external hosted micro front-end.</p>
```

Comment:
I do not get what is the intent of saying this. Maybe it would be interesting to say that this example behaves like a single page app.

Reply:
How about: `This micro front-end behaves like a single page application.`

Reply:
That is better I think.